hphp/hack/doc/vision_docs/tracking_effects_and_coeffects.md

```diff
@@ -891,7 +891,7 @@ also Haskell and others (E-lang, EffScript).
 
 * ScalaSpec: [Scala Language Specification](https://www.scala-lang.org/files/archive/spec/2.11/) (M. Odersky et al.)
 * **ScalaDays’15**: [**Scala** - where it came from, where it is going](https://www.slideshare.net/Odersky/scala-days-san-francisco-45917092) (M. Odersky), pages 44-46
-* **POPL’18**: [Simplicitly](https://dl.acm.org/doi/pdf/10.1145/3158130) (M. Odersky et al.)
```

Comment:
It's an unusual title, but it is the name of this paper: https://dl.acm.org/doi/pdf/10.1145/3158130